packages/e2e-tests/src/specs/editor/media/insertMovVideo.js

```diff
@@ -54,29 +55,27 @@ describe('Handling .mov files', () => {
     });
 
     await deleteMedia(fileNameNoExt);
+    await toggleVideoOptimization();
```

Comment:
Nit: consider doing this in an `afterEach`/`afterAll` to guarantee cleanup even if the test fails before that. Otherwise following tests might fail due to the checkbox not being correctly toggled.